lib/lte_link_control/lte_lc.c

```diff
@@ -1612,7 +1612,17 @@ int lte_lc_conn_eval_params_get(struct lte_lc_conn_eval_params *params)
 
 int lte_lc_modem_events_enable(void)
 {
-	return nrf_modem_at_printf(AT_MDMEV_ENABLE) ? -EFAULT : 0;
+	/* First try to enable both warning and informational type events, which is only supported
+	 * by mfw > v1.3.x.
```

Comment:
We have used "modem firmware versions" rather than "mfw" in many places.
And I think it would be better to say `>=` than `>` especially because > 1.3.x is a bit harder to understand as people could start to look 1.4.x etc. while it's only really supported in 2.0.0.

```suggestion
	 * by modem firmware versions >= 2.0.0.
```

Reply:
I agree, it's clearer to say that it's supported by `>= 2.0.0`. Or another possibility is to remove the version number completely, I'm not sure it's mandatory to say that in code comments. It could be just `newer modem firmware versions`, IMHO.

Reply:
But I don't see what harm exact version number brings when we know it clearly.

Reply:
Fixed